client/reader/following/main.jsx

```diff
@@ -79,7 +92,17 @@ const FollowingStream = ( props ) => {
 				/>
 			</CompactCard>
 			<BlankSuggestions suggestions={ suggestionList } />
-			<SectionHeader label={ translate( 'Followed sites' ) }>
+			<SectionHeader label={ translate( 'Followed Sites' ) }>
+				{ config.isEnabled( 'reader/seen-posts' ) && ! props.hasUnseen && (
+					<Button compact onClick={ markAllAsUnSeen }>
+						{ translate( 'Mark All as Unseen' ) }
```

Comment:
This is inconsistent with the `Mark all as unseen` above. Could you change this to a single way of spelling it so that we can reduce the number of translations?

Reply:
They should be fixed in the next PR https://github.com/Automattic/wp-calypso/pull/42355/files thanks for the headsup